app/src/components/MobileLeftExpand.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
```

Comment:
Please stop creating new components inside this folder. Move it elsewhere, please